File: microservices/audit/audit-dto/src/main/java/dto/audit/DataAuditEventEnum.java
package dto.audit;

import javax.xml.bind.annotation.XmlEnum;
import javax.xml.bind.annotation.XmlEnumValue;

@XmlEnum
public enum DataAuditEventEnum {
    @XmlEnumValue("CREATED")
    CREATED,

    @XmlEnumValue("MODIFIED")
    MODIFIED,

    @XmlEnumValue("REMOVED")
    REMOVED;
}

File: net-core/microservices/audit/audit.dto/DataAuditEventEnum.cs
using Newtonsoft.Json;

namespace audit.dto
{
    public enum DataAuditEventEnum
    {
        [JsonProperty(PropertyName = "CREATED")]
        Created = 0,
        
        [JsonProperty(PropertyName = "MODIFIED")]
        Modified = 1,
        
        [JsonProperty(PropertyName = "REMOVED")]
        Removed = 2
    }
}